pkgs/servers/ldap/lldap/default.nix

```diff
@@ -31,21 +32,28 @@ let
       hash = "sha256-UbkpnPvDoTaioB0g1HBvRczqUEuPlVuH1gIVBU07RpY=";
     };
 
-  };
-
-  frontend = rustPlatform.buildRustPackage (commonDerivationAttrs // {
-    pname = commonDerivationAttrs.pname + "-frontend";
-
     # `Cargo.lock` has git dependencies, meaning can't use `cargoHash`
     cargoLock = {
-      lockFile = ./Cargo.original.lock;
+      lockFile = ./Cargo.lock;
       outputHashes = {
         "lber-0.4.1" = "sha256-2rGTpg8puIAXggX9rEbXPdirfetNOHWfFc80xqzPMT4=";
         "opaque-ke-0.6.1" = "sha256-99gaDv7eIcYChmvOKQ4yXuaGVzo2Q6BcgSQOzsLF+fM=";
         "yew_form-0.1.8" = "sha256-1n9C7NiFfTjbmc9B5bDEnz7ZpYJo9ZT8/dioRXJ65hc=";
       };
     };
 
+    patches = [
+      (fetchpatch {
+        url = "https://github.com/lldap/lldap/pull/947.patch";
```

Comment:
This isn't a stable URL — the hash will break if I have to change the PR.  For unmerged PRs, we have to include the patch in Nixpkgs.

Reply:
Still hoping that we get a release soon, but for now I've made your patch URL stable by referring to commits. No need to include that patch file.